Source/Core/DolphinQt/Debugger/CodeViewWidget.cpp

```diff
@@ -455,14 +456,13 @@ void CodeViewWidget::OnReplaceInstruction()
   if (!read_result.valid)
     return;
 
-  bool good;
-  QString name = QInputDialog::getText(
-      this, tr("Change instruction"), tr("New instruction:"), QLineEdit::Normal,
-      QStringLiteral("%1").arg(read_result.hex, 8, 16, QLatin1Char('0')), &good);
+  u32 code = read_result.hex;
 
-  u32 code = name.toUInt(&good, 16);
+  // Show instruction edit dialog
+  InstructionDialog* instr_dialog = new InstructionDialog(this, addr, code);
```

Comment:
Unless I am overlooking something, this InstructionDialog will only be deleted once the CodeViewWidget is closed; for subsequent instructions, another dialog is constructed without disposing the old one.

I would suggest either setting `WA_DeleteOnClose` for it or creating it on the stack instead.
The latter makes more sense IMO, given that it is a modal.

While I don't see how this would lead to a crash, this might be the problem you're having.

Reply:
Sure, that makes more sense, changed.
Still didn't resolve the crashing. Is anybody else running Win10 and can confirm the crashing?